README.md

```diff
@@ -82,32 +78,28 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[My recent works into]**
+- **[My recent works intro]**
 - **[Recent works]**
 - **[About me section]**
+- **[Animations and transitions on buttons and project cards]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link]([(https://github.com/torobucci/Portfolio-finish-mobile-version)])
+- [Live Demo Link] (https://torobucci.github.io/Portfolio-finish-mobile-version/)
```

Comment:
[OPTIONAL] I would advise you to use it like this `[Live Demo Link](https://torobucci.github.io/Portfolio-finish-mobile-version/)` this will make the link address not show but still there.